### PR TITLE
Map 1403 CSC-UI NPM dependency updates

### PR DESCRIPTION
**Pull request by GurnankCheema**
some visible changes:
1. phase banner colour change see [this](https://design-system.service.gov.uk/components/phase-banner/#wcag-do-not-cover-content) 
2. no yellow+red border in error summary. Now just red only. [this](https://design-system.service.gov.uk/components/error-summary/)
3. the underline below links is thinner even if the text is large.This is the same as on other dps pages like the 'add casenote' link on prisoner profile

### CODE REVIEW

File: assets/js/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import * as govukFrontend from 'node_modules/govuk-frontend/dist/govuk/all'
```
Comment (GurnankCheema) [inline assets/js/index.js:1]: assets/js/govukFrontendInit.js deleted and replaced by index.js to matche the typescript template's approach for initialising.

File: assets/scss/application-ie8.scss [hunk 2]
```diff
@@ -1,7 +0,0 @@
-$govuk-global-styles: true;
```
Comment (GurnankCheema) [inline assets/scss/application-ie8.scss:1]: Govuk-fronend library no longer supports ie8 see 
https://frontend.design-system.service.gov.uk/changes-to-govuk-frontend-v5/#main-changes-to-v5

File: server/utils/nunjucksSetup.ts [hunk 3]
```diff
@@ -31,10 +31,8 @@ export default function nunjucksSetup(app: express.Express, applicationInfo: App
   const njkEnv = nunjucks.configure(
     [
       path.join(__dirname, '../../server/views'),
-      'node_modules/govuk-frontend/',
-      'node_modules/govuk-frontend/components/',
+      'node_modules/govuk-frontend/dist/',
```
Comment (GurnankCheema) [inline server/utils/nunjucksSetup.ts:34]: now same as typescript template project
https://github.com/ministryofjustice/hmpps-template-typescript/blob/eb86ef136745d50101ef8f902f5c3476a5f4ceb8/server/utils/nunjucksSetup.ts#L29

File: server/views/cellMove/selectCell.njk [hunk 4]
```diff
@@ -190,11 +190,4 @@
 {% endblock %}
 
 {% block bodyEnd %}
-  {{ super() }}
```
Comment (GurnankCheema) [inline server/views/cellMove/selectCell.njk:193]: code block removed as this page doesn't apply any sorting

File: server/middleware/setUpStaticResources.ts [hunk 5]
```diff
@@ -17,8 +17,8 @@ export default function setUpStaticResources(): Router {
     '/assets',
     '/assets/stylesheets',
     '/assets/js',
-    '/node_modules/govuk-frontend/govuk/assets',
-    '/node_modules/govuk-frontend',
+    '/node_modules/govuk-frontend/dist/govuk/assets',
```
Comment (GurnankCheema) [inline server/middleware/setUpStaticResources.ts:20]: to match https://github.com/ministryofjustice/hmpps-template-typescript/blob/eb86ef136745d50101ef8f902f5c3476a5f4ceb8/server/middleware/setUpStaticResources.ts#L17

File: package.json [hunk 6]
```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "prepare": "husky install",
     "copy-views": "cp -R server/views dist/server/",
-    "compile-sass": "sass --quiet-deps --no-source-map --load-path=node_modules/govuk-frontend --load-path=node_modules/@ministryofjustice/frontend --load-path=node_modules/govuk_frontend_toolkit/stylesheets --load-path=node_modules/govuk-elements-sass/public/sass --load-path=. assets/scss/application.scss:./assets/stylesheets/application.css assets/scss/application-ie8.scss:./assets/stylesheets/application-ie8.css --style compressed",
```
Comment (GurnankCheema) [inline package.json:10]: ie8 scss files have been removed

**Review by Mjwillis — DISMISSED** [review]
LGTM

**Comment by GurnankCheema** [comment]
@Michael Willis Thank you for PR-ing. I'm going to wait until everything else  in the pipeline is fully tested and deployed before I merge as it may be difficult to debug any errors if everything goes in together.

**Pull request merged**